code/game/machinery/computer/communications.dm

```diff
@@ -432,6 +432,7 @@
 		authenticated = 2
 	to_chat(user, "<span class='danger'>You scramble the communication routing circuits!</span>")
 	playsound(src, 'sound/machines/terminal_alert.ogg', 50, FALSE)
+	SSshuttle.shuttle_purchase_requirements_met |= SHUTTLE_UNLOCK_DISCOINFERNO
```

Comment:
This isn't the way to do this. Not only is this only for the inferno, but it also doesn't include the other shuttles.

The correct way to do this is to have a flag on the alternate shuttle datum that marks whether it is emag only or not.

From there, only show those in the shuttle menu when the console is emagged (check how it checks for whether to send messages to the syndicate or CentCom), and only allow them to purchased at this point as well.

Reply:
The force push made this outdated--but it's not.